content/SCALECLIReference/Storage/CLIDisk.md

```diff
@@ -11,7 +11,624 @@ tags:
+
+Where:
+* *attribute* is the attribute you want to filter in the query.
+
+{{< expand "Command Example" "v" >}}
+```
+storage disk query name
++---------+
+| name    |
++---------+
+| sdb     |
+| sda     |
+| nvme0n1 |
+| sdc     |
+| sdd     |
+| sdm     |
+| sdn     |
+| sde     |
+| sdf     |
+| sdg     |
+| sdh     |
+| sdi     |
+| sdj     |
+| sdk     |
+| sdl     |
++---------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Resize Command
+
+The `resize` command allows you to resize disks to a specific amount of gigabytes.
+
+{{< expand "Resizing Disks">}}
+
+#### Description
+The `resize` command has three properties. They are `disks`, `sync`, and `raise_error`.
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns completion percentages when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>storage disk resize disks={"name":"<i>diskname</i>,<i>diskname</i>", "size":"<i>number</i>"} sync=<i>true/false</i> raise_error=<i>true/false</i><code>
+
+Where:
+* *diskname* is the name of the disk you want to resize. Separate multiple disks with commas
+* *number* is the size (in gigabytes) you want to resize the disk to.
+* *true/false* in the `sync` property enables or disables synchronizing the new size of the disk(s) with the database cache. The default is true.
+* *true/false* in the `raise_error` property sets whether the disk(s) raise a CallError upon failure (true) or will only log errors (false). The default is false.
+
+{{< expand "Command Example" "v" >}}
+```
+storage disk resize disks={"name":"sda,sdb", "size":"894"} sync=true raise_error=false
+[0%] ...
+[100%] ...
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Retaste Command
+
+The `retaste` command forces the system to re-read specified disks and update their data.
+
+{{< expand "Retasting Disks">}}
+
+#### Description
+The `retaste` command requires you to specify which disks you want to retaste.
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns completion percentages when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>storage disk retaste disks=<i>name</i>,<i>name</i><code>
+
+Where *name* is the name of a disk you want to retaste. Separate each disk name with a comma.
+
+{{< expand "Command Example" "v" >}}
+````
+storage disk retaste disks=sdl
+[85%] Retasting disks...
+[95%] Waiting for disk events to settle...
+[100%] Retasting disks done...
+SUCCESS
+````
+{{< /expand >}}
+{{< /expand >}}
+
+### Smart_Attributes Command
+
+The `smart_attributes` command returns S.M.A.R.T. attributes values for specified disk.
+
+{{< hint type=note >}}
+Only devices with the ATA bus type support S.M.A.R.T. attributes.
+{{< /hint >}}
+
+{{< expand "Viewing S.M.A.R.T. Attributes">}}
+
+#### Description
+The `smart_attributes` command requires you to specify which disk you want to view S.M.A.R.T. attributes for.
+Enter the command string, then press <kbd>Enter</kbd>.
```

Comment:
Tell them how to enter the property argument (w/wo double-quoted values if that apples). Follow the how with this sentence.